openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCardNew/CommentCard.component.tsx

```diff
@@ -0,0 +1,150 @@
+ */
+import { Tooltip, Typography } from 'antd';
+import classNames from 'classnames';
+import { compare } from 'fast-json-patch';
+import React, { useCallback, useMemo, useState } from 'react';
+import { Thread } from '../../../generated/entity/feed/thread';
+import {
+  formatDateTime,
+  getRelativeTime,
+} from '../../../utils/date-time/DateTimeUtils';
+import {
+  getFrontEndFormat,
+  MarkdownToHTMLConverter,
+} from '../../../utils/FeedUtils';
+import ProfilePicture from '../../common/ProfilePicture/ProfilePicture';
+import FeedCardFooterNew from '../ActivityFeedCardV2/FeedCardFooter/FeedCardFooterNew';
+import ActivityFeedEditor from '../ActivityFeedEditor/ActivityFeedEditorNew';
+import { useActivityFeedProvider } from '../ActivityFeedProvider/ActivityFeedProvider';
+import ActivityFeedActions from '../Shared/ActivityFeedActions';
+
+const { Text } = Typography;
+
+interface CommentCardInterface {
+  feed: Thread;
+  post: any;
+  isLastReply: boolean;
+}
+
+const CommentCard = ({ feed, post, isLastReply }: CommentCardInterface) => {
+  const { updateFeed } = useActivityFeedProvider();
+  const [isHovered, setIsHovered] = useState(false);
+  const [isEditPost, setIsEditPost] = useState<boolean>(false);
+  const [postMessage, setPostMessage] = useState<string>('');
+  const seperator = '.';
+
+  const onEditPost = () => {
+    setIsEditPost(!isEditPost);
+  };
+
+  const onUpdate = (message: string) => {
+    const updatedPost = { ...feed, message };
+    const patch = compare(feed, updatedPost);
+    updateFeed(feed.id, post.id, false, patch);
+    setIsEditPost(!isEditPost);
+  };
+
+  const handleSave = useCallback(() => {
+    onUpdate?.(postMessage ?? '');
+  }, [onUpdate, postMessage]);
+
+  const getDefaultValue = (defaultMessage: string) => {
+    return MarkdownToHTMLConverter.makeHtml(getFrontEndFormat(defaultMessage));
+  };
+
+  const feedBodyRender = useMemo(() => {
+    if (isEditPost) {
+      return (
+        <ActivityFeedEditor
+          focused
+          className="mb-8 reply-feed-editor"
+          defaultValue={getDefaultValue(post.message)}
+          editorClass="is_edit_post"
+          onSave={handleSave}
+          onTextChange={(message) => setPostMessage(message)}
+        />
+      );
+    }
+
+    return null;
+  }, [isEditPost, postMessage, handleSave]);
+
+  function stripHtml(html: any) {
+    const tempDiv = document.createElement('div');
+    tempDiv.innerHTML = html;
+
+    return tempDiv.innerText || tempDiv.textContent;
+  }
+
+  return (
+    <div
+      className={classNames('d-flex justify-start relative reply-card', {
+        'reply-card-border-bottom': !isLastReply,
+      })}
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}>
+      <div className="profile-picture m-r-xs">
+        <ProfilePicture
+          avatarType="outlined"
+          key={feed.id}
+          name={feed.updatedBy!}
+          size={32}
+        />
+      </div>
```

Comment:
I see we are adding div just to add some margin?

Reply:
removed